documentation/images/plantuml/agent/flows/multi_ap_policy_config_request.puml

```diff
@@ -0,0 +1,14 @@
+@startuml
+controller->backhaul_manager: send_cmdu\n(MULTI_AP_POLICY_CONFIG_REQUEST_MESSAGE)
+
+backhaul_manager->backhaul_manager: handle_multi_ap_policy_config_request()\nStore AP Metrics Reporting Interval\nvalue for periodic AP metrics reporting.\nForward CMDU to all slaves whose RUID is\nspecified in Metric Reporting Policy TLV\n
+backhaul_manager->son_slave_wlan0: forward_cmdu\n(MULTI_AP_POLICY_CONFIG_REQUEST_MESSAGE)
+backhaul_manager->son_slave_wlan2: forward_cmdu\n(MULTI_AP_POLICY_CONFIG_REQUEST_MESSAGE)
```

Comment:
(nitpick) Although technically inaccurate, for the AP Metrics Query flow we have written it down as if it were synchronous, i.e. first the ACK, then the full flow for wlan0, then the full flow for wlan2. We felt that this was visually more easy to understand.

Reply:
I was going to propose you the modification of AP Metrics Query flow to make it more accurate!

Reply:
Either way works for me. It's a matter of taste I guess. Consistency would probably be good. But unified agent will anyway need to change this again, so it's not very important I supposed. Maybe having two different examples is actually good - when someone new reads it, they can decide which option they like most :-)